docs/en/Clients-Go.md

```diff
@@ -39,18 +39,51 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"log"
+	"strings"
 
-	alluxio "github.com/Alluxio/alluxio-go"
-	"github.com/Alluxio/alluxio-go/option"
+	alluxio "github.com/alluxio/alluxio-go"
+	"github.com/alluxio/alluxio-go/option"
 )
 
 func main() {
-	fs := alluxio.NewClient(<proxy-host>, <proxy-port>, <timeout>)
-	ok, err := fs.Exists("/test_path", &option.Exists{})
+	fs := alluxio.NewClient(<proxy host>, <proxy port (39999 by default)>, <timeout>)
```

Comment:
Maybe just use the defaults and comment in the detailed instructions since the rest of the code works?

Reply:
good suggestion, will submit another PR